### ISSUE TITLE
Verify that Hatchet settings/configs for MozFest still works

### ISSUE THREAD

**Issue by mmmavis** [MozillaFoundation/mofo-devops#501]
Hey @cadecairos 

MozFest's Call for Proposals opens on June 1 and I'm currently working on getting all the updates implemented on the MozFest site. Last year we had transaction emails sent to people after they submitted their proposals. This year we would like to do the same (note that I need to do some code cleanup still). I see on Heroku there are a few configs related to Hatchet, could you help verify that those are still valid? Thanks!

/cc @edrushka

**Comment by cadecairos** [comment]
@mmmavis I noticed that the app was using credentials for a user that had far more permissions than it needed to send messages to sawmill. I created a new AWS user and key pair for the mozillafestival.org server and updated the credentials. 

TLDR: it should work perfectly, but if not, lets open a new issue to troubleshoot it.
